arch/x86/core/intel64/userspace.S

```diff
@@ -108,6 +108,8 @@ z_x86_syscall_entry_stub:
 
 	sti			/* re-enable interrupts */
 
+	nop
```

Comment:
Safe, too.  It's comparing the contents of RAX (which must not change if we are interrupted) with a constant.